spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -66,16 +65,17 @@ public class TaskTemplate implements TaskOperations {
 
 	private static final String EXECUTIONS_CURRENT_RELATION_VERSION = "1.7.0";
 
-	private static final String VALIDATION_RELATION_VERSION = "1.7.0";
```

Comment:
Why is this deleted? Please add a comment.

Reply:
I restored that.

Reply:
I don't see it.